python/tests/unit/arcticdb/version_store/test_resample.py

```diff
@@ -90,6 +112,83 @@ def test_resampling_duplicated_index_value_on_segment_boundary(lmdb_version_stor
     )
 
 
+class TestResamplingBucketInsideSegment:
+
+    def test_all_buckets_have_values(self, lmdb_version_store_v1):
+        lib = lmdb_version_store_v1
+        sym = "test_inner_buckets_are_empty"
+        idx = pd.DatetimeIndex([
+            dt.datetime(2023, 12, 7, 23, 59, 47, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 48, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 49, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 50, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 51, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 52, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 53, 500000),
+            dt.datetime(2023, 12, 7, 23, 59, 55, 500000)
+        ])
+        df = pd.DataFrame({"mid": [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0]}, index=idx)
+        lib.write(sym, df)
+    
+        date_range = (dt.datetime(2023, 12, 7, 23, 59, 48), dt.datetime(2023, 12, 7, 23, 59, 52))
+        generic_sparse_resample_test(lib, sym, 's', {'high': ('mid', 'max')}, date_range=date_range)
+
+    @pytest.mark.parametrize("closed", ("left", "right"))
+    def test_first_bucket_is_empy(self, lmdb_version_store_v1, closed):
```

Comment:
Is this name accurate? I think the first bucket will only be empty if the closed boundary is `right`?

Reply:
In both cases the date range will cut out `dt.datetime(2023, 12, 7, 23, 59, 48, 342000)` so I think it doesn't matter if it's left or right.